lib/dial/transport.go

```diff
@@ -0,0 +1,38 @@
+package dial
+
+import "fmt"
+
+const (
+	transportDefault string = ""
+	transportPlain   string = "plain"
+	transportSsl     string = "ssl"
+)
+
+//go:generate stringer -type Transport -trimprefix Transport -linecomment
```

Comment:
Please, add a test for the String() function to make sure that everything work fine:

https://github.com/tarantool/go-discovery/blob/master/eventtype_test.go

You could also add tarantport_string.go to the commit. The stringer-generated code usually included into a commit.